quinn/src/builders.rs

```diff
@@ -174,6 +174,15 @@ impl ServerConfigBuilder {
         Ok(self)
     }
 
+    /// Whether to accept 0-RTT data from clients
+    ///
+    /// 0-RTT allows application data to be transmitted without waiting for the handshake to
+    /// complete, but might be replayed by attackers.
+    pub fn zero_rtt(&mut self, enabled: bool) -> &mut Self {
+        Arc::make_mut(&mut self.config.crypto).max_early_data_size = enabled as u32 * 0xffffffff;
+        self
+    }
+
```

Comment:
This is redundant as of 94a37e399e4d68df42db1c79a4a93829e4994df5; we always declare support for 0-RTT in TLS, and leave it up to the application to decide whether to actually use it.

Reply:
Removed.